Automattic/wp-calypso
# Copy Site: Add domain step to copy site flow.

#### Proposed Changes

Add domain step to copy site flow

#### Testing Instructions



1. Navigate to `/sites`
2. Find a site with a `Business` plan. (It will not work for `eCommerce` plans due to auto transfer. Note that `eCommerce` plans will not be available for copying after [this](https://github.com/Automattic/wp-calypso/pull/72262) and D98470-code merged)
3. Ensure that you can select a domain as shown:
![domain_step_copy_site](https://user-images.githubusercontent.com/497103/213188465-e2443bda-d5ef-4b38-b772-c9933f9464a7.png)
4. Select a domain, and checkout
5. Make sure that your site has been copied successfully.


#### Pre-merge Checklist





Fixes https://github.com/Automattic/dotcom-forge/issues/1515
